src/v2/Components/Authentication/Views/LoginForm.tsx

```diff
@@ -172,6 +172,11 @@ export class LoginForm extends Component<FormProps, LoginFormState> {
                       this.props.onFacebookLogin?.(e)
                     })
                   }}
+                  onGoogleLogin={e => {
+                    this.setState({ isSocialSignUp: true }, () => {
+                      this.props.onGoogleLogin?.(e)
+                    })
+                  }}
```

Comment:
@jacobherrington - since this is a new behavior/feature, we should add a test before merge